tests/test_jx/test_edge_1.py

```diff
@@ -436,15 +436,15 @@ def test_union_values(self):
                 "data": [
                     {"a": "x", "c": [2, 3]},
                     {"a": "y", "c": [4, 5, 6]},
-                    {"a": NULL, "c": [7, 8]}
+                    {"a": NULL, "c": [8, 7]}
```

Comment:
Made these changes in tests to ask if the order should matter in UNION of values.